content/cookbook/tokens/non-fungible-tokens.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Non-Fungible Tokens (NFTs)
+sidebarSortOrder: 17
+description:
+  "Learn how to create, mint, and transfer non-fungible tokens (NFTs) on Solana,
+  using metaplex, token-metadata, umi, candy machine v2."
+---
+
+## Candy Machine v2
```

Comment:
This doc has to be removed, since we are removing candy machine code for now as my discussion with jacob. I committed this accidently, thanks for the nice catch.